momepy/functional/_elements.py

```diff
@@ -252,9 +271,14 @@ def _tess(ix, poly, blg, threshold, shrink, segment, enclosure_id):
             return_input=False,
             as_gdf=True,
         )
+        if to_simplify:
+            tess.geometry = coverage_simplify(tess.geometry, 1e-1)
```

Comment:
```suggestion
            tess.geometry = coverage_simplify(tess.geometry, 1e-1, simplify_boundary=False)
```
Otherwise you break contiguity and match with streets